pkg/tenantctl/controller.go

```diff
@@ -184,21 +185,24 @@ func (c *Controller) syncHandler(key string) error {
 
 	tenant := origTenant.DeepCopy()
 
-	c.logPrintln(tenant, "started reconciliation")
-	defer c.logPrintln(&api.Tenant{ObjectMeta: *tenant.ObjectMeta.DeepCopy()}, "finished reconciliation")
+	klog.V(1).Infof(c.formatLog(tenant, "started reconciliation"))
+	start := time.Now()
+	klog.V(1).Infof(c.formatLog(&api.Tenant{ObjectMeta: *tenant.ObjectMeta.DeepCopy()}, "finished reconciliation"))
+	elapsed := time.Since(start)
+	log.Printf("Binomial took %s", elapsed)
```

Comment:
`log` usage?

Reply:
ah this one i didnt want to commit sorry